schema/cassandra/temporal/schema.cql

```diff
@@ -126,6 +126,9 @@ CREATE TABLE cluster_metadata (
   metadata_partition      int,
   immutable_data          blob,
   immutable_data_encoding text,
```

Comment:
according to the current PR, these 2 should be removed?

Reply:
At the moment they were needed, but I refactored the code and now they were removed.